Sources/NIOCore/FileHandle.swift

```diff
@@ -19,6 +19,12 @@ import Darwin
 import Glibc
 #endif
 
+#if os(Windows)
+public typealias POSIXFileMode = CInt
+#else
+public typealias POSIXFileMode = mode_t
```

Comment:
Nit: We need to prefix this with `NIO`, otherwise this is a good name.

Reply:
Yeah, that is a good idea ... should've realized that.